ui/packages/app/web/src/App.tsx

```diff
@@ -31,26 +33,28 @@ function getBasename() {
 }
 
 const App = () => {
-  const createStore = useCreateStore();
+  let persistor = persistStore(store);
```

Comment:
Shouldn't this be outside the App component? I guess, otherwise it would create a new store every time the App component renders? 🤔 

Or we could even move it to `./store` file and export the persistor from there as well? I saw this here: https://github.com/rt2zz/redux-persist#basic-usage

Reply:
great catch! I'll move it to `./store` - agree that's the best place for it!